### PR TITLE
feat: faucet

### PR DESCRIPTION
**Pull request by rjchow**

### CODE REVIEW

File: src/faucet/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      keyId: process.env.KMS_KEY_ID,
+    };
+
+    const provider = ethers.getDefaultProvider("ropsten");
+    let signer = new AwsKmsSigner(kmsCredentials);
+    signer = signer.connect(provider);
+    const receiver = event.pathParameters.walletAddress;
+    if (!utils.isAddress(receiver)) throw new Error("Invalid wallet address");
+    const signerBalance = ethers.utils.formatEther(await signer.getBalance());
+    if (signerBalance < 1) throw new Error("Insufficient Funds in wallet.");
```
Comment (rjchow) [inline src/faucet/index.js]: maybe `Oops! Faucet has ran dry, please inform the Open-Attestation team.` would be less confusing for the user?

File: src/faucet/index.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({
+        address: receiver,
+        txhash: transfer.hash,
+        amount: "1",
+      }),
+    });
+  } catch (e) {
+    callback(null, {
+      statusCode: 400,
```
Comment (rjchow) [inline src/faucet/index.js]: 400 should be returned when user input is invalid, e.g invalid wallet address etc
500 should be returned when our side has issues, like no more eth or whatever other server errors

Comment (isaackps) [reply src/faucet/index.js]: ok will update it

File: src/faucet/index.js (hunk 1 above)
Comment (isaackps) [reply src/faucet/index.js]: ok

File: src/faucet/index.js [hunk 3]
```diff
@@ -35,7 +40,7 @@ const handleFaucet = async (event, _context, callback) => {
     });
   } catch (e) {
     callback(null, {
-      statusCode: 400,
+      statusCode: 500,
```
Comment (rjchow) [inline src/faucet/index.js]: what about the 400 errors?

**Pull request closed without merging**